sql/src/main/java/org/apache/druid/sql/calcite/rel/Windowing.java

```diff
@@ -88,9 +88,9 @@ public class Windowing
   private static final ImmutableMap<String, ProcessorMaker> KNOWN_WINDOW_FNS = ImmutableMap
       .<String, ProcessorMaker>builder()
       .put("LAG", (agg) ->
-          new WindowOffsetProcessor(agg.getColumn(0), agg.getOutputName(), -agg.getConstantInt(1)))
+          new WindowOffsetProcessor(agg.getColumn(0), agg.getOutputName(), -agg.getConstantInt(1, 1)))
```

Comment:
I think, If we are implying that LAG(dimension) without the offset always means 1, we should also document it somewhere

Reply:
actually; I don't really see any existing documentation for LEAD/LAG; and because I don't see any either for windowing - I think that should be added separately along with LEAD/LAG